apps/web/src/components/content/privacy-wrapper.tsx

```diff
@@ -95,31 +94,50 @@ export function PrivacyWrapper({
       provider: provider,
     })
 
-    if (showIframe && provider === ExternalProvider.Vocaroo) return null
-    if (isTwingle && showIframe) return null
```

Comment:
Probably, the placeholder had to be disabled in these cases because if would show through when the embedded content was rendered on top using `position: absolute`. But I did not test it. 
I think it is save to remove because `renderPlaceholder` in this PR is only called when `showContent == false`